### PR TITLE
feat: renames package to aind_data_transfer

### PR DESCRIPTION
**Pull request by jtyoung84**

### CODE REVIEW

**Comment by jtyoung84** [comment]
I'll rename the repo to aind-data-transfer after this is merged

**Comment by jtyoung84** [comment]
I'll also re-enable the github version bump action after this is merged

File: scripts/gcs_upload.py [hunk 1]
```diff
@@ -164,7 +164,7 @@ def _parse_cp_failures(log_path):
         log_path (str): path to the log file created by gsutil,
                         e.g., gsutil cp -L log_path.log ...
     Returns:
-        a list of filepaths that resulted in an error during transfer
+        a list of filepaths that resulted in an error during aind_data_transfer
```
Comment (carshadi) [inline scripts/gcs_upload.py:167]: string replaced in comment

**Review by carshadi — APPROVED** [review]

File: scripts/gcs_upload.py (hunk 1 above)
Comment (jtyoung84) [reply scripts/gcs_upload.py]: Good catch. I updated it.

**Pull request merged**